package.nls.json

```diff
@@ -422,7 +422,7 @@
     "AWS.s3.downloadFile.saveButton": "Download",
     "AWS.s3.downloadFile.error.general": "Failed to download file {0}",
     "AWS.s3.fileViewer.button.dismiss": "Don't show this again",
-    "AWS.s3.fileViewer.button.help": "Help",
+    "AWs.s3.fileViewer.button.ok": "Ok",
```

Comment:
Nit: capitalize "OK". Nit #2: If this isn't already a generic string (see bottom), you should make it generic.